CHANGELOG.md

```diff
@@ -35,6 +57,10 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Added user consent content reload screen
 - Public: When `photoCaptureFallback` option is disabled in the face step, the requested variant is video and browser does not support MediaRecorder, attempt to redirect to the cross-device flow or display the unsupported browser error.
 - Internal: Refactor for better reusability of video capture components.
+- Internal: Added support for testing across multiple browsers.
+- Internal: Added support for visual regression testing using Percy.
+- Internal: Resolved inaccurate reporting on test automation failures.
+- Internal: Added ability to execute/ignore tests via the use of tags/regex.
```

Comment:
These feel like they should be under current release. @SherBhachuOnFido these are unreleased changes still, right?